ea_restaurant_data_loader/lib/entities/app_client_scope.ex

```diff
@@ -5,14 +5,15 @@ defmodule EaRestaurantDataLoader.AppClientScope do
 
   schema "app_clients_scopes" do
     field(:scope, :string)
+    field(:app_client_id, :integer)
     field(:entity_status, :string)
     field(:created_date, :naive_datetime)
     field(:updated_date, :naive_datetime)
     field(:created_by, :integer)
     field(:updated_by, :integer)
     belongs_to(:created_by_user, User, source: :created_by, define_field: false)
     belongs_to(:updated_by_user, User, source: :updated_by, define_field: false)
-    belongs_to(:app_client_id, AppClient)
+    belongs_to(:app_client_id_id, AppClient, source: :app_client_id, define_field: false)
```

Comment:
Ditto

Reply:
Also, I think you need to add a `has_many` relationship to AppClient. An AppClient should have many scopes

Reply:
No. The relationship  is not necessary as the scopes are inside a list. it is just one field for app_client with a list of scopes.